mcc/odin/odin/snapshot_list_test.go

```diff
@@ -74,6 +86,22 @@ var listSnapshotsCases = []listSnapshotsCase{
 		snapshots:  []*rds.DBSnapshot{exampleSnapshot1},
 		instanceID: "",
 	},
+	// Two instances, two snapshots
+	{
+		testCase: testCase{
+			expected: []*rds.DBSnapshot{
+				exampleSnapshot1,
+				exampleSnapshot2,
+			},
+			expectedError: "",
+		},
+		name: "Two instances two snapshots",
+		snapshots: []*rds.DBSnapshot{
+			exampleSnapshot1,
+			exampleSnapshot2,
+		},
+		instanceID: "",
+	},
```

Comment:
New test case :)
In order to clarify, its creating another `[]*rds.DBSnapshot` with the same content, to avoid confusion when using references.